eng/pipelines/libraries/enterprise/linux.yml

```diff
@@ -0,0 +1,73 @@
+
+# Disable pipeline for ordinary pushes to the branches
+trigger: none
+
+# To reduce load on the pipeline, enable it only for PRs that affect critical networking code
+pr:
+  branches:
+    include:
+    - master
+    - release/*.*
+
+  paths:
+    include:
+      - src/libraries/Common/src/System/Net/*
```

Comment:
You also need to include `eng/pipelines/libraries/enterprise/*` to make sure when this file is changed you get the necessary coverage.

Reply:
Thx. I'll add that in my follow-up PR.

Reply:
Fixed in #843